dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventsExportController.java

```diff
@@ -296,9 +296,40 @@ ResponseEntity<InputStreamResource> getEventDataValueImage(
   }
 
   @GetMapping("/{uid}/changeLog")
-  List<EventChangeLog> getEventByUid(@OpenApi.Param({UID.class, Event.class}) @PathVariable UID uid)
+  Page<ObjectNode> getEventByUid(
```

Comment:
```suggestion
  Page<ObjectNode> getEventChangeLogByUid(
```

OpenAPI cannot handle overloaded methods and this method is not returning events.